src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -18,6 +18,9 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
 
   <PropertyGroup>
     <_DefaultJavaSourceJarPattern>**\*-source.jar;**\*-sources.jar;**\*-src.jar</_DefaultJavaSourceJarPattern>
+    <!-- Ignore Resources and files in SubDirectories -->
+    <_DefaultLibraryItemExcludes Condition=" '$(Configuration)' == 'Debug' " >$(DefaultItemExcludes);**/bin/Release/**;**/obj/Release/**</_DefaultLibraryItemExcludes>
+    <_DefaultLibraryItemExcludes Condition=" '$(Configuration)' == 'Release' ">$(DefaultItemExcludes);**/bin/Debug/**;**/obj/Debug/**</_DefaultLibraryItemExcludes>
```

Comment:
I'm not sure this is actually valid, because you can name `$(Configuration)` anything you want. A lot of times people have  names like `AppStore`, `GooglePlay`, `Amazon`, etc.

Will it still work if we only specify `bin` and `obj` here?

Also, does it do any file I/O if you put `**/somedir/` in an `Exclude`?